src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -217,6 +216,9 @@ public static bool TryFloat(FormattingInfo formatInfo, IRContext irContext, Form
                     }
 
                     break;
+
+                default:
+                    return false;

```

Comment:
[minor] Not really needed - it would return false below as the result won't be set